controls/V-72433.rb

```diff
@@ -77,15 +77,20 @@
 to include \"ocsp_on\"."
   tag "fix_id": "F-78785r3_fix"
 
-  describe command("grep cert_policy /etc/pam_pkcs11/pam_pkcs11.conf") do
-    its('stdout') { should include 'ocsp_on' }
-  end if smart_card_status.eql?('enabled')
 
-  describe command("grep cert_policy /etc/pam_pkcs11/pam_pkcs11.conf | wc -l") do
-    its('stdout.strip.to_i') { should cmp >= 3 }
-  end if smart_card_status.eql?('enabled')
-
-  describe "The system is not smartcard enabled" do
-    skip "The system is not using Smartcards / PIVs to fulfil the MFA requirement, this control is Not Applicable."
-  end if !smart_card_status.eql?('enabled')
+  if smart_card_status.eql?('enabled')
+    cert_policy_lines = file('/etc/pam_pkcs11/pam_pkcs11.conf').content.lines.grep(%r{^(?!.+#).*cert_policy}i)
```

Comment:
This will fail if `/etc/pam_pkcs11/pam_pkcs11.conf` does not exist.

Probably need a test just before all of this to ensure that the file exists at all since, if it doesn't, the whole things fails anyway.

Reply:
I believe this is resolved by 92e53b5